src/components/JoinUs/OpenPositions.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import { Row, CompensatedCol, CompensatedRow } from '../grid'
+import styled from 'styled-components'
+import JobsByLocation from '../jobsByLocation'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2, H5 } from '../Typography'
+import { FirstColumn, SecondColumn, Section } from './elements'
+import ExternalAnchor from '../Common/ExternalAnchor'
+import Hr from '../Common/Hr'
+import StyledLink from '../styledLink'
+
+const Line = styled(Hr)`
+  width: ${remcalc(60)};
+`
+
+const JobLinkWrapper = styled.div`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+`
+
+const JobLink = styled(ExternalAnchor)`
+  flex: 1 0 90px;
+`
+
+const Job = ({ text, hostedUrl, categories: { commitment } }) => (
+  <CompensatedCol as="li" width={[1, 1, 1, 1, 4 / 12, 3 / 12]}>
+    <JobLinkWrapper>
+      <JobLink href={hostedUrl}>
+        <Padding top={1} bottom={0.5}>
+          <Paragraph noMargin>{text}</Paragraph>
+          <Paragraph noMargin muted>
+            {commitment}
+          </Paragraph>
+        </Padding>
+      </JobLink>
+      <Line />
+    </JobLinkWrapper>
+  </CompensatedCol>
+)
+
+const DirectApplication = ({ title, text }) => (
```

Comment:
Can we use the existing `TalkToUs` component that's in Common?

Reply:
where's that component?